lib/routes.jsx

```diff
@@ -50,7 +51,8 @@ exports.generateStatic = function(url, cb) {
 };
 
 exports.run = function(location, el) {
-  ga.initialize();
+  var options = { debug: false };
```

Comment:
Oh wait, that debug thing is quite verbose. Perhaps we should just add a `GA_DEBUG` env var, and if it is `on` then we set that flag to true.